addons/mail/static/src/core/common/primary_variables.scss

```diff
@@ -31,5 +31,6 @@ $o-mail-Discuss-headerHeight: 48px !default;
 }
 
 .o-mail-systrayFullscreenDropdownMenu {
```

Comment:
Not in diff but this looks miss-placed, it's not a primary variable.

Reply:
Yeah conceptually it should be in `core.scss` (and all of its content, actually).

IIRC some variables need to be in `primary_variables` and not in `core.scss` (and vice-versa) due to some SCSS dependencies based on parsing of (S)CSS files at page load.